person.rb

```diff
@@ -0,0 +1,45 @@
+require_relative 'nameable'
+
+# Person represents an individual with attributes such as name, age, and parent_permission.
+class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id
+
+  # Initializes a new Person object.
+  # age: The age of the person.
+  # name: The name of the person (default is 'Unknown').
+  # parent_permission: A boolean indicating if the person has parent permission (default is true).
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super
+    @id = generate_id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] Great job so far @gersonhiggins There is just a minor change I will suggest you fix and you'll be on your way to the next milestone :wink: 
- [ ] I noticed that when I tried to run the `ruby.rb` file and test if you correctly decorated your classes and all your methods work as expected, I got an error. I have attached a screenshot below. So kindly fix this so the next reviewer can confirm if the right output is being returned :+1: 

- Error Message
![image](https://github.com/gersonhiggins/oop-ruby/assets/102699958/98089b8b-f1bc-463d-ad1b-62054c7fb4f4)